config/config.exs

```diff
@@ -32,6 +32,11 @@ config :guardian, Guardian,
   secret_key: "some_secret_key",
   serializer: SheriffExampleApp.GuardianSerializer
 
+config :sheriff, Sheriff,
```

Comment:
The basic config + setting the resource key to be guardians default, instead of `current_user` as sheriff defaults to